### PR TITLE
Sidebar Refactored

### PR DESCRIPTION
**Pull request by aryanvaid2108**
**Steps to test:** 
1. Login using admin credentials : 
email: admin@admin.com
pass: admin123

Expected Result: New Side Bar

### CODE REVIEW

File: frontend/src/Components/Sidebar/SidebarData.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        icon: <SiGooglecalendar />,
+        cName: 'nav-text'
+    },
+    {
+        title:'Files',
+        path:'/files',
+        icon: < AiFillFilePdf/>,
+        cName: 'nav-text'
+    },
+    {},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},
```
Comment (bubblemans) [inline frontend/src/Components/Sidebar/SidebarData.js:48]: If this is to separate the logout button, I do not see logout button

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,61 +1,26 @@
...
-        }
-        <ContextStore.Provider value={{ CurrentUser: [user,setUser], SideBar: [showSidebar, setShowSidebar]}}>
-            <Route path="/" exact component={withRouter(LoginForm)}/>
-            <Route path="/register" component={RegisterPage}/>
-            <Route path="/home" component={withRouter(HomePage)}/>
-        </ContextStore.Provider>
-      </React.Fragment>
-      )}
-      />
+        <Sidebar />
```
Comment (bubblemans) [inline frontend/src/App.js:16]: Why do we have sidebar in login page?

**Review by bubblemans — APPROVED** [review]
Look at the feedbacks below.

File: frontend/src/Components/Sidebar/SidebarData.js (hunk 1 above)
Comment (aryanvaid2108) [reply frontend/src/Components/Sidebar/SidebarData.js:48]: It is at the bottom

**Pull request merged**